### ISSUE TITLE
unproper daemon call

### ISSUE THREAD

**Issue by michaelrsweet** [apple/cups#746]
Version: 1.1.14 or older
CUPS.org User: tlazcala.yahoo

 Dear Sir:

 I don't know if the following is a bug but it took to
me a lot of time to fixed it up. May be this notice it
is not know and can it save others' time.
  I installed CUPS package in a woody-debian with ekplp backend to manage a printer Epson c62. It worked well from the beginning, but I needed to update to Sarge-debian to work with another package, and things with CUPS crashed. Printer didn't work at all [Pdwaa,
for short]. So I re-installed CUPS with Sarge, and changed the backend with the help of gimp-print. But Printer, ..., Ok. [Pdwaa]. I read manual again, I tried to configure CUPS again and ... nothing. I walked back, deleted all the system and re-installed Woody, again, with ancient CUPS package and ekplp backend and, ... nothing. This was a real surprise. OK. after several tests, debug level in confs files, man pages and script readings. I realized that ekpd wasn't initilized at PC startup though cupsd was, and I could'nt initialize it manually neither.
Daemons cupsd and ekpd had symlinks in /etc/rcS that I didn't remember there was the very first time I installed the packages. I deleted those symlinks to call cupsd and ekpd at runlevel 2 as their first call, and things worked up again. No more [Pwdaa]. 
 In conclusion cupsd allows to ekpd to work only if it is restarted at runlevel 2 and then ekpd it is called. As I didn't write symlinks to cupsd in /etc/rc0, the install-package did. So I think some parameter is wrong in the *.deb package who forces cupsys to start at rcS (i.e. runlevel 0).
 Because this I wrote this letter to you.
